include/sys/time.h

```diff
@@ -349,6 +349,40 @@ int getitimer(int which, FAR struct itimerval *value);
 int setitimer(int which, FAR const struct itimerval *value,
               FAR struct itimerval *ovalue);
 
+/****************************************************************************
+ * Name: utimes
+ *
+ * Description:
+ * The utimes() function shall set the access and modification times of the
+ * file pointed to by the path argument to the value of the times argument.
+ * utimes() function allows time specifications accurate to the microsecond.
+
+ * For utimes(), the times argument is an array of timeval structures. The
+ * first array member represents the date and time of last access, and the
+ * second member represents the date and time of last modification. The times
+ * in the timeval structure are measured in seconds and microseconds since
+ * the Epoch, although rounding toward the nearest second may occur.
+
+ * If the times argument is a null pointer, the access and modification times
+ * of the file shall be set to the current time. The effective user ID of the
+ * process shall match the owner of the file, has write access to the file or
+ * appropriate privileges to use this call in this manner. Upon completion,
+ * utimes() shall mark the time of the last file status change, st_ctime, for
+ * update.
+ *
+ * Input Parameters:
+ *   path  - Specifies the file to be modified
+ *   times - Specifies the time value to set
+ *
+ * Returned Value:
+ *   Upon successful completion, 0 shall be returned. Otherwise, -1 shall be
+ *   returned and errno shall be set to indicate the error, and the file
+ *   times shall not be affected.
+ *
+ ****************************************************************************/
+
+int utimes(const char *path, const struct timeval times[2]);
```

Comment:
```suggestion
int utimes(FAR const char *path, const struct timeval times[2]);
```
Also from the commit message "but skip the implemenation because VFS layer doesn't support the time/mode change yet:"
The time change is supported, it's only implemented by a couple of file systems though, FAT and NXFFS I believe.

Reply:
utimes require to change the file time not query:
https://pubs.opengroup.org/onlinepubs/009695399/functions/utimes.html
VFS support query but not change yet.

Reply:
> Also from the commit message "but skip the implemenation because VFS layer doesn't support the time/mode change yet:"
> The time change is supported, it's only implemented by a couple of file systems though, FAT and NXFFS I believe.

Could you point me what VFS API can change the file time? I can't find one:(.

Reply:
Sorry there is no VFS API exported.  FAT and NXFFS change the stat struct which is then requested by stat() and family.

Reply:
@xiaoxiang781216 Could you add the suggested `FAR` so we can merge this one?